packages/components/src/localStorage/index.ts

```diff
@@ -0,0 +1,26 @@
+// import AsyncStorage from '@react-native-community/async-storage';
+
```

Comment:
Name of file should not start from a small letter. It should be LocalStorage instead of localStorage.

Reply:
done